.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "vendor/llhttp"]
     path = vendor/llhttp
     url = https://github.com/nodejs/llhttp.git
-    branch = v8.x
+    branch = main
```

Comment:
There's no v9 branch. So, it looks like we'll be best just leaving it on main, and being sure to checkout a particular tag when updating.